File: Command/PHP/Player.php
<?php
  class Player {
    protected $state;

    public function __construct() {
      $this->state = "";
    }

    public function playing() {
      $this->state = "playing";
    }

    public function pause() {
      $this->state = "pause";
    }

    public function stop() {
      $this->state = "stop";
    }

    public function getState() {
      return $this->state;
    }
  }

File: Command/Go/command/Player.go
package command

type Player struct {
  state string
}

func NewPlayer() *Player {
  return &Player{""}
}

func (this *Player) Playing() {
  this.state = "playing"
}

func (this *Player) Pause() {
  this.state = "pause"
}

func (this *Player) Stop() {
  this.state = "stop"
}

func (this *Player) GetState() string {
  return this.state
}
